src/isar_turtlebot/config/default.ini

```diff
@@ -5,7 +5,7 @@ image_filetype = jpg
 thermal_image_filetype = png
 
 [ros_bridge]
-host = localhost
+host = noetic
```

Comment:
Should we remove this?

Reply:
It has been removed. Unless you mean removing it for that specific commit?

Reply:
I mean for that commit, yes.

Reply:
Should be correct now.